README.md

```diff
@@ -39,13 +45,27 @@ The modules in the monorepo are organized into two categories:
 - Install dependencies: `pnpm i`
 
 - Start server:
+  
   `cd apps/api`
+  
+  `touch .env`
+
+  Add `.env` values
+
```

Comment:
consider using `cp .env.schema .env` to make a copy of the default env schema.
User then has the default schema as a base and can adjust accordingly.

Reply:
Amazing, had a feeling this could be improved